docs/administration/authentication/microsoft-azure-ad.md

```diff
@@ -61,6 +61,43 @@ Restart the NetBox services so that the new configuration takes effect. This is
 sudo systemctl restart netbox
 ```
 
+## Group Assignment
+
+If you want NetBox to assign groups based on Azure AD groups, then some additonal configuration is needed. Enter the following configuration parameters in `configuration.py`, substituting your own values:
+
+```python
+SOCIAL_AUTH_AZUREAD_OAUTH2_RESOURCE = 'https://graph.microsoft.com/'
+SOCIAL_AUTH_PIPELINE = (
+    'social_core.pipeline.social_auth.social_details',
+    'social_core.pipeline.social_auth.social_uid',
+    'social_core.pipeline.social_auth.social_user',
+    'social_core.pipeline.user.get_username',
+    'social_core.pipeline.social_auth.associate_by_email',
+    'social_core.pipeline.user.create_user',
+    'social_core.pipeline.social_auth.associate_user',
+    'netbox.authentication.user_default_groups_handler',
+    'social_core.pipeline.social_auth.load_extra_data',
+    'social_core.pipeline.user.user_details',
+    'netbox.authentication.azuread_map_groups',
+)
+SOCIAL_AUTH_AZUREAD_MAP_GROUP_PERMS = True
+
+# Define special user types using groups. Exercise great caution when assigning superuser status.
+SOCIAL_AUTH_AZUREAD_USER_FLAGS_BY_GROUP = {
```

Comment:
I recall we talked about the parameter naming strategy last week, but I'm having second thoughts. I worry that introducing our own paramters prefixed with `SOCIAL_AUTH_AZUREAD` may cause confusion among users expecting to see it the `python-social-auth` docs. I don't have a better suggestion though; open to other opinions.

Reply:
Why not prefix with `NETBOX_`?

Reply:
changed to REMOTE_AUTH_BACKEND_ and auto-imported everything starting with this as we will probably be adding more so can group them all together.